internal/graph/graph.go

```diff
@@ -171,7 +171,9 @@ func Referrers(ctx context.Context, target content.ReadOnlyGraphStorage, desc oc
 		default:
 			continue
 		}
-		if node.ArtifactType != "" && (artifactType == "" || artifactType == node.ArtifactType) {
+		if artifactType == "" || artifactType == node.ArtifactType {
+			// artifactType is allowed to be empty & empty will pass the filtering by default
```

Comment:
This comment is requested for re-wording as the term `artifactType` is ambiguous since it be the `artifactType` of a manifest or the filter `artifactType`. The term `filtering` is also too generic and is requested to be re-worded so that other folks know it referrers to referrers filtering.

Reply:
Try not to use `&`. Use `and` instead for formal writing.